homeassistant/components/climate/generic_thermostat.py

```diff
@@ -30,6 +30,8 @@
 
 DEFAULT_TOLERANCE = 0.3
 DEFAULT_NAME = 'Generic Thermostat'
+DEFAULT_AWAY_TEMP_COOL = 30
```

Comment:
Why do you need `away` temperatures for both heating and cooling? Setting a heating away temperature together with `ac_mode = True` does not make sense, and vice versa. I guess that if the user sets `ac_mode = True`, he should known that the `away` temperature is used for cooling rather than for heating. I would just call it `away_temperature`.

Reply:
Just read your answer to the comment by @MartinHjelmare , that you want to implement heating and cooling with the same entity. I think that's a good idea (and then you of course need both settings); however I think either both or none of it should be part of this PR, i.e., either also implement the "both heating and cooling logic", or postpone splitting the away temperature into two until a later PR.

Reply:
I have to admit that I didn't think much about this. My initial intention was to fix the operation_mode and add away_mode functionality seen in another PR. I mostly reused what was developed there (#11081). 
I think you are right, for now it should only be one away temperature and I will correct this until heating/cooling logic is added.